nf_core/lint/merge_markers.py

```diff
@@ -13,6 +13,17 @@ def merge_markers(self):
     This test looks for remaining merge markers in the code, e.g.:
     >>>>>>> or <<<<<<<
 
+    .. tip:: You can choose to ignore this test tests by creating a file called
```

Comment:
```suggestion
    .. tip:: You can choose to ignore this lint tests by creating a file called
```
don't we have that file now by default?

Reply:
true! it's in the template. I will change the verb by "editing the file"

Reply:
I edited your comment because I can't modify the same line from two commends and I wanted to keep the "co-authored" 😬